spring-cloud-skipper-server-core/pom.xml

```diff
@@ -14,6 +14,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <version>3.1.0</version>
```

Comment:
Can we put all new deps to dep management in parent pom? Same in `spring-cloud-skipper-server`